### PR TITLE
fix pagination

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

File: ovs-service/src/main/java/org/dcsa/ovs/service/VesselScheduleService.java [hunk 1]
```diff
@@ -46,18 +53,9 @@ public static class ServiceSchedulesFilters {
...
-    Cursor c =
-        paginator.parseRequest(
-            request,
-            new CursorDefaults(
-                requestFilters.limit,
-                new Cursor.SortBy(Sort.Direction.DESC, Service_.CARRIER_SERVICE_CODE)));
+  public VesselScheduleTOPage findAll(Cursor cursor, final ServiceSchedulesFilters requestFilters) {
 
-    return serviceRepository
+    Page<org.dcsa.ovs.persistence.entity.Service> page = serviceRepository
```
[the remark this answers is not in the document]
Comment (FinnAtAsseco) [reply ovs-service/src/main/java/org/dcsa/ovs/service/VesselScheduleService.java]: Wont be needed with the new pagination classes

File: ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java [hunk 2]
```diff
@@ -21,42 +28,49 @@
...
-      // TODO: use the vesselIMONumber validator from core
-      @Size(max = 7) @RequestParam(required = false) String vesselIMONumber,
-      @Size(max = 35) @RequestParam(required = false) String vesselName,
-      @Size(max = 50) @RequestParam(required = false) String voyageNumber,
-      @RequestParam(required = false) String universalVoyageReference,
-      @Size(max = 5) @RequestParam(value = "UNLocationCode", required = false)
+    @Size(max = 5) @RequestParam(required = false) String carrierServiceCode,
+    @Size(max = 8) @RequestParam(required = false) String universalServiceReference,
+    // TODO: use the vesselIMONumber validator from core
+    @Size(max = 7) @RequestParam(required = false) String vesselIMONumber,
```
Comment (mp-assco) [inline ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java:39]:
```suggestion
    @ValidVesselIMONumber(allowNull = true) @RequestParam(required = false) String vesselIMONumber,
```

File: ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java [hunk 3]
```diff
@@ -1,14 +1,21 @@
 package org.dcsa.ovs.controller;
 
 import lombok.RequiredArgsConstructor;
+import org.dcsa.ovs.persistence.entity.Service_;
 import org.dcsa.ovs.service.VesselScheduleService;
 import org.dcsa.ovs.transferobjects.VesselScheduleTO;
```
[the remark this answers is not in the document]
Comment (FinnAtAsseco) [reply ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java:6]: unused imports removed

File: ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java [hunk 4]
```diff
@@ -21,42 +29,49 @@
...
-      // TODO: use the vesselIMONumber validator from core
-      @Size(max = 7) @RequestParam(required = false) String vesselIMONumber,
-      @Size(max = 35) @RequestParam(required = false) String vesselName,
-      @Size(max = 50) @RequestParam(required = false) String voyageNumber,
-      @RequestParam(required = false) String universalVoyageReference,
-      @Size(max = 5) @RequestParam(value = "UNLocationCode", required = false)
+    @Size(max = 5) @RequestParam(required = false) String carrierServiceCode,
+    @Size(max = 8) @RequestParam(required = false) String universalServiceReference,
+    // TODO: use the vesselIMONumber validator from core
+    @ValidVesselIMONumber(allowNull = true) @RequestParam(required = false) String vesselIMONumber,
```
Comment (nt-as) [inline ovs-service/src/main/java/org/dcsa/ovs/controller/ServiceScheduleController.java:40]: We are now using the validator, so the comment can go?

**Comment by FinnAtAsseco** [comment]
Included in align entities

**Pull request closed without merging**